packages/webapp/src/containers/Finances/index.jsx

```diff
@@ -38,21 +38,23 @@ import { downloadFinanceReport } from './saga';
 import FinancesCarrousel from '../../components/Finances/FinancesCarrousel';
```

Comment:
Relevant changes in this file as well, changing how KPIs are calculated and displaying the filter button